pkg/skaffold/docker/image.go

```diff
@@ -156,18 +156,36 @@ func (l *localDaemon) ConfigFile(ctx context.Context, image string) (*v1.ConfigF
 	return cfg, nil
 }
 
+func authConfig(ctx context.Context) map[string]types.AuthConfig {
+	auth := make(chan map[string]types.AuthConfig)
+
+	go func() {
+		// Like `docker build`, we ignore the errors
+		// See https://github.com/docker/cli/blob/75c1bb1f33d7cedbaf48404597d5bf9818199480/cli/command/image/build.go#L364
+		authConfigs, _ := DefaultAuthHelper.GetAllAuthConfigs()
```

Comment:
nit: should we pass context into the function `DefaultAuthHelper.GetAllAuthConfigs` and handle it there?

Reply:
I'll do that in another PR. The code is more complicated. But you are right that it makes more sense.